core/citrus-base/src/main/java/org/citrusframework/report/LoggingReporter.java

```diff
@@ -45,27 +47,45 @@
  */
 public class LoggingReporter extends AbstractTestReporter implements MessageListener, TestSuiteListener, TestListener, TestActionListener {
 
-    /** Inbound message logger */
+    /**
+     * Logger
+     */
+    private static Logger logger = LoggerFactory.getLogger(LoggingReporter.class);
+
+    /**
+     * Inbound message logger
+     */
     private static Logger inboundMessageLogger = LoggerFactory.getLogger("Logger.Message_IN");
 
-    /** The inbound message logger used when the reporter is enabled */
+    /**
+     * The inbound message logger used when the reporter is enabled
+     */
     private static final Logger enabledInboundMessageLogger = inboundMessageLogger;
 
-    /** Outbound message logger */
+    /**
+     * Outbound message logger
+     */
     private static Logger outboundMessageLogger = LoggerFactory.getLogger("Logger.Message_OUT");
 
-    /** The inbound message logger used when the reporter is enabled */
+    /**
+     * The inbound message logger used when the reporter is enabled
+     */
     private static final Logger enabledOutboundMessageLogger = outboundMessageLogger;
 
-    /** Logger */
-    private static Logger logger = LoggerFactory.getLogger(LoggingReporter.class);
-
-    /** The standard logger used when the reporter is enabled */
+    /**
+     * The standard logger used when the reporter is enabled
+     */
     private static final Logger enabledLog = logger;
 
-    /** A {@link org.slf4j.helpers.NOPLogger} used in case the reporter is not enabled. */
+    /**
+     * A {@link org.slf4j.helpers.NOPLogger} used in case the reporter is not enabled.
+     */
     private static final Logger noOpLogger = new NOPLoggerFactory().getLogger(LoggingReporter.class.getName());
 
+    private static String formatDurationString(TestCase test) {
+        return nonNull(test.getTestResult()) ? " (" + test.getTestResult().getDuration().toString() + ") " : "";
```

Comment:
Duration to string creates a special format like this: PT12.000000012S, is this intentional?